src/serac/physics/tests/solid.cpp

```diff
@@ -322,11 +322,11 @@ void functional_parameterized_solid_test(double expected_disp_norm)
 
   // add some nonexistent body forces / tractions to check that
   // these parameterized versions compile and run without error
-  solid_solver.addBodyForce(DependsOn<0>{}, [](const auto& x, double /*t*/, auto /* bulk */) { return x * 0.0; });
+  // solid_solver.addBodyForce(DependsOn<0>{}, [](const auto& x, double /*t*/, auto /* bulk */) { return x * 0.0; });
```

Comment:
Is this planning on being reenabled or is it dead?

Reply:
these should be uncommented, thanks for reminding me!